rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessReturnTagRector/Fixture/remove_null.php.inc

```diff
@@ -7,7 +7,7 @@ class RemoveNull
     /**
      * @return null
      */
-    function foo()
+    function foo(): ?int
     {
         return null;
     }
```

Comment:
These should be kept

Reply:
sure https://github.com/rectorphp/rector-src/pull/5350/commits/ede7425008c805852fbd532a12392887df662f8e,

Reply:
I mean this fixture file should be untouched.

Reply:
The reason why `@return null` actually valid here, is that it sub type of defined type, which make more correct result of what user want to.

The typed can be enforced by interface, so can't change, but the only way to enforce it is via `@return null`.

Reply:
Remove `null` only make sense when it exactly different type ( which already handled ), eg:

```php
/** @return null */
function foo(): int
```